waspc/examples/crud-testing/src/server/auth.ts

```diff
@@ -9,21 +9,22 @@ import prisma from '@wasp/dbClient.js'
 import { CustomSignup } from '@wasp/actions/types'
 import { sanitizeAndSerializeProviderData } from '@wasp/auth/utils.js'
 
-export const fields = defineAdditionalSignupFields({
-  address: (data) => {
-    const AddressSchema = z
-      .string({
-        required_error: 'Address is required',
-        invalid_type_error: 'Address must be a string',
-      })
-      .min(10, 'Address must be at least 10 characters long')
-    const result = AddressSchema.safeParse(data.address)
-    if (result.success === false) {
-      throw new Error(result.error.issues[0].message)
-    }
-    return result.data
-  },
-})
+export const getUserFields = () =>
+  defineUserFields({
```

Comment:
Not sure if I missed something: so this `defineUserFields` is a new thing?

Reply:
Yep, we renamed it from `defineAdditionalSignupFields` to `defineUserFields` and now we expect the developer to define a function that returns the `fields` instead of just `fields`